Makefile

```diff
@@ -27,7 +27,7 @@ static ?=       ## Enable static linking
 O := .build
 SOURCES := $(shell find src -name '*.cr')
 SPEC_SOURCES := $(shell find spec -name '*.cr')
-FLAGS := $(if $(release),--release )$(if $(stats),--stats )$(if $(progress),--progress )$(if $(threads),--threads $(threads) )$(if $(debug),-d )$(if $(static),--static )
+FLAGS := $(if $(release),--release )$(if $(stats),--stats )$(if $(progress),--progress )$(if $(threads),--threads $(threads) )$(if $(debug),-d )$(if $(static),--static )$(if $(CRYSTAL_FLAGS),$(CRYSTAL_FLAGS) )
```

Comment:
Splitting `$(CRYSTAL_FLAGS)` changes into a separate line for sake of clarity might be a good thing here, WDYT @aurimasniekis?

Reply:
If you think it's better than yes I could do it. But personally, I do not think many people go into Makefile

Reply:
I think so, yes. ATM it's the longest line in the whole file and chis change would make it even longer, no nice.